tests/unit_tests/test_wave_06.py

```diff
@@ -22,23 +22,25 @@ def test_get_items_by_category():
     assert item_a in items
     assert item_c in items
 
-@pytest.mark.skip
+#@pytest.mark.skip
 def test_get_no_matching_items_by_category():
+    #arrange
     item_a = Clothing()
     item_b = Item()
     item_c = Decor()
     vendor = Vendor(
         inventory=[item_a, item_b, item_c]
     )
-
+    #act
     items = vendor.get_by_category("Electronics")
-
-    raise Exception("Complete this test according to comments below.")
+    #assert
+    assert len(items) == 0
+    assert not items
```

Comment:
I would say that it would be okay to remove the second assert statement. Mainly because the first line implicitly checks for this already.
```suggestion
    assert len(items) == 0
```